src/pages/patientView/PatientViewGenePanelModal/PatientViewGenePanelModal.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { GenePanel } from 'shared/api/generated/CBioPortalAPI';
+import { observer } from 'mobx-react';
+import GenePanelModal from 'shared/components/GenePanelModal/GenePanelModal';
+import GenesList from './GenesList';
+
+interface IGenePanelModalProps {
+    genePanelCache: GenePanel;
+    panelName: string;
```

Comment:
is it necessary to pass this in separately? cant it be inferred from the `GenePanel` object?